Notify/Notify/Notify/AppShell.xaml.cs

```diff
@@ -35,6 +43,35 @@ public AppShell()
             }
         }
 
+        private void ConnetivityChangedHandeler(object sender, ConnectivityChangedEventArgs e)  // TODO: move into Notify.Android
+        {
+            var connectivityManager = (ConnectivityManager)Android.App.Application.Context.GetSystemService(Context.ConnectivityService);
+            var capabilities = connectivityManager.GetNetworkCapabilities(connectivityManager.ActiveNetwork);
+
+            if (capabilities.HasTransport(TransportType.Wifi))
+            {
+                var wifiManager = (WifiManager)Android.App.Application.Context.GetSystemService(Context.WifiService);
+                string ssid = wifiManager.ConnectionInfo.SSID;
+        
+                if (ssid == my_ssid)
+                {
+                    notificationManager.SendNotification("Wifi - Push Notification", 
+                        "You have just connected to your wifi network!");
+                    Debug.WriteLine("Wifi - Push Notification. \nSSID: " + ssid);
+                }
+                else
+                {
+                    notificationManager.SendNotification("Error with ssid",
+                    "SSID: " + ssid + "  my_ssid: " + my_ssid);
+                }
+            }
+            else
+            {
+                notificationManager.SendNotification("Disconnected from Wifi network!","");
+                Debug.WriteLine("Disconnected from WIFI network!");
```

Comment:
change `WIFI` word to be the same in all places, so it will be consistent.
Some places were `Wifi`

Reply:
Done.
It's 'Wifi' - if it's at the beginning of a sentence and 'wifi' if not/